modules/vector-search/pages/vector-search-index-architecture.adoc

```diff
@@ -0,0 +1,152 @@
+
+Vector Search uses a combination of IVF and flat indexes for data that contains between 1000 and 9999 vectors.
+For even larger datasets, Vector Search uses IVF indexes with <<scalar-quant,>>.
+
+IVF creates partitions called Voronoi cells in an index. 
+The total number of cells is the *nlist* parameter. 
+
+Every cell has a centroid.
+Every vector in the processed dataset is assigned to a cell that corresponds to its nearest centroid. 
+
+In an IVF index, a Vector Search first tries to find the cell that the query vector belongs to.
+After it knows the cell to search, Vector Search uses another algorithm to find out the exact vector that's closest to the query vector in that cell. 
+
+The result of an IVF index search can be less accurate, as the nearest vector to a query vector can be in a different cell than the chosen cell.
+You can increase accuracy by changing the *nprobe* parameter when you xref:fine-tune-vector-search.adoc[fine tune your Vector Search queries].
+
+Larger IVF indexes automatically train to learn the data distribution of your vectors, and the centroids of cells in your dataset.
+The training data helps to encode and compress the vectors in your index with <<scalar-quant,>>.
+All training occurs during building and merging <<segments,>>.
+
+IVF indexes that also use flat indexing automatically train to determine the centroids of cells, but still use exact vector comparisons within each cell.
+Training still occurs while building and merging <<segments,>>. 
+
+[#scalar-quant]
+==== Scalar Quantization 
+
+Vector Search uses scalar quantization on large datasets to reduce the size of your indexes. 
+
+Scalar quantization is an important data compression technique that turns the floating point values that could be present in a large vector into low-dimensional integers.
+For example, a float32 value could be reduced to an int8 value. 
+
+Scalar quantization in Vector Search does not have a significant effect on the recall, or accuracy, of query results on large datasets.
+
+Vector Search uses both 8bit and 4bit scalar quantization for indexes, based on your xref:search:child-field-options-reference.adoc#optimized[Optimized For] setting.
+
+== Search Request Processing 
+
+The Search Service uses a scatter-gather process for running all Search queries, when there are multiple nodes in the cluster running the Search Service.
+
+The Search Service node that receives the Search request is assigned as the coordinating node.
+Using https://grpc.io/[gRPC^], the coordinating node scatters the request to all other partitions for the Search or Vector Search index in the request across other nodes.
+The coordinating node applies filters to the results received from the other partitions, and returns the final result set.
+
+Results are scored, and based on the xref:search:search-request-params.adoc#sort[Sort Object] provided in the Search request, returned in a list.
```

Comment:
Should mention tf-idf and vector distance scores are summed during hybrid search and the user can influence the scoring with the `boost` setting.